src/main/User/UserController.java

```diff
@@ -293,6 +293,24 @@ public static JSONObject mergeJSON(
         UploadedFile file = ctx.uploadedFile("file");
         log.info(username + " is attempting to upload a profile picture");
         UploadPfpService serv = new UploadPfpService(db, username, file, fileName);
+        // TODO: enable UploadFileService:
+        //        UploadFileService serv =
+        //            new UploadFileService(
+        //                db,
+        //                username,
+        //                null,
```

Comment:
I would like to move away from null constructor arguments- could we maybe just use Optional?